src/Steps.js

```diff
@@ -0,0 +1,149 @@
+
+class FirstStep {
+    render () {
+        const title = document.createElement('h1')
+        title.innerText = 'Formularz kontaktowy'
+
+        let stepTitle = document.createElement('h3')
+        stepTitle.innerText = 'Krok 1'
+
+        const button1 = document.createElement('button')
+        button1.innerText = 'Jestem klientem'
+
+        const button2 = document.createElement('button')
+        button2.innerText = 'Jestem kandydatem'
+
+        formularz.wrapperHTMLElement.innerHTML = ''
+        formularz.wrapperHTMLElement.appendChild(title)
+        formularz.wrapperHTMLElement.appendChild(stepTitle)
+        formularz.wrapperHTMLElement.appendChild(button1)
+        formularz.wrapperHTMLElement.appendChild(button2)
+
+        button1.addEventListener('click', () => {
+            formularz.activeStep = new SecondStep('klient')
+            formularz.render()
+        })
+
+        button2.addEventListener('click', () => {
+            formularz.activeStep = new SecondStep('kandydat')
+            formularz.render()
+        })
+    }
+}
+
+class SecondStep {
+    constructor (a) {
+        this.type = a
+    }
+    render () {
+        const title = document.createElement('h1')
+        title.innerText = 'Formularz kontaktowy'
+
+        let stepTitle = document.createElement('h3')
+        stepTitle.innerText = 'Krok 2'
+
+        const formWrapper = document.createElement('div')
+        let input1, input2, input3
+
+        if (this.type === 'klient') {
+            console.log('klient')
+
+            input1 = document.createElement('input')
+            input1.placeholder = 'Nazwa firmy'
+            formWrapper.appendChild(input1)
+
+            input2 = document.createElement('textarea')
+            input2.placeholder = 'Treść wiadomości'
+            formWrapper.appendChild(input2)
+        }
+        else if (this.type === 'kandydat') {
+            // console.log('kandydat')
+            input1 = document.createElement('input')
+            input1.placeholder = 'Imię i nazwisko'
+            formWrapper.appendChild(input1)
+
+            input2 = document.createElement('textarea')
+            input2.placeholder = 'Treść wiadomości'
+            formWrapper.appendChild(input2)
+
+            fetch('http://pliki.rekiny.wroclaw.pl/rekrutacja/stanowiska.json')
```

Comment:
Metody `fetch` oraz cała logika odpowiedzialna za pobieranie danych powinna znajdować się w osobnym pliku dla lepszej czytelności. Podobnie odnośniki do endpointów mogłyby znajdować się w osobnych zmiennych np: const URL_API = 'http://pliki.rekiny.wroclaw.pl/rekrutacja/stanowiska.json' po to aby w przypadku ich modyfikacji, robić to w jednym miejscu bez potrzeby szukania w projekcie.